src/Orchard.Web/Core/Common/Views/CommonMetadataOwner.cshtml

```diff
@@ -11,4 +11,4 @@
     // owner isn't really who last modified this, is it?
     IUser owner = commonPart.As<CommonPart>() == null ? null : commonPart.As<CommonPart>().Owner;
 }
-@T("By {0}", owner == null ? T("unknown").ToString() : Convert.ToString(Html.ItemDisplayText(owner)))
\ No newline at end of file
+@T("By {0}", owner == null ? T("unknown").ToString() : Convert.ToString(Html.ItemDisplayText(owner, false)))
```

Comment:
The solution is to remove the `ToString` calls from this line. No to remove encoding.

```
@T("By {0}", owner == null ? T("unknown") : Html.ItemDisplayText(owner, false)) 

```